code/game/objects/items/stacks/sheets/mineral.dm

```diff
@@ -22,6 +22,11 @@ Mineral Sheets
 /obj/item/stack/sheet/mineral
 	icon = 'icons/obj/mining.dmi'
 
+/obj/item/stack/sheet/mineral/Initialize()
+	..()
+	pixel_x = rand(0,4)-4
```

Comment:
I think mine was a rand minus another rand(I don't remember) but yeah this is just equivalent to `pixel_x = rand(-4, 0)`